client/src/plugins/search/Types.ts

```diff
@@ -1,7 +1,7 @@
 import { Option } from '../../components/customSelector/Types';
 import { searchKeywordsType } from '../../consts/Milvus';
-import { DataType, DataTypeEnum } from '../collections/Types';
-import { IndexView } from '../schema/Types';
+import { DataType, DataTypeEnum } from 'insight_src/pages/collections/Types';
```

Comment:
why we need to use insight_src instead of relative path? 
when should we use it?  And will it affect the webpack building if we only set in the tsconfig?

Please make some comments or write in readme.md

Reply:
Use `insight_src` alias instead of relative path for:
1. Plugins should be independent.
2. For using/developing, plugins need some components/hooks inside of Insight, it will be easy to use alias instead of relative path.
It will affect the webpack building, because of `react-app-rewire-alias` in `config-overrides.js`.
Readme updates is ongoing.